### ISSUE TITLE
0.3.0 does not build

### ISSUE THREAD

**Issue by ed-aka-yaboi** [jerry73204/realsense-rust#12]
I just ran rustup update to sync to stable-x86_64-unknown-linux-gnu and am on ubuntu 18.04

my cargo.toml file looks like so 

```toml
[package]
name = "random"
version = "0.1.0"
authors = ["someAuthor <someAuthorgmail.com>"]
edition = "2018"

# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html

[dependencies]
realsense-rust = "0.3.0"
```

my main.rs is as follows:
```rust
fn main() {
    println!("hi");
}
```

When I run cargo clippy the build fails for the following reasons:
```
Checking realsense-rust v0.3.0
error[E0432]: unresolved import `image::ConvertBuffer`
 --> /home/sigkill/.cargo/registry/src/github.com-1ecc6299db9ec823/realsense-rust-0.3.0/src/base.rs:4:24
  |
4 | use image::{Bgr, Bgra, ConvertBuffer, DynamicImage, ImageBuffer, Luma, Rgb, Rgba};
  |                        ^^^^^^^^^^^^^ no `ConvertBuffer` in the root

error[E0599]: no method named `convert` found for struct `image::buffer_::ImageBuffer<image::color::Bgr<u8>, &[u8]>` in the current scope
    --> /home/sigkill/.cargo/registry/src/github.com-1ecc6299db9ec823/realsense-rust-0.3.0/src/base.rs:112:68
     |
112  |             Rs2Image::Bgr8(image) => DynamicImage::ImageBgr8(image.convert()),
     |                                                                    ^^^^^^^ method not found in `image::buffer_::ImageBuffer<image::color::Bgr<u8>, &[u8]>`
     | 
    ::: /home/sigkill/.cargo/registry/src/github.com-1ecc6299db9ec823/image-0.23.6/./src/buffer.rs:1059:8
     |
1059 |     fn convert(&self) -> T;
     |        -------
     |        |
     |        the method is available for `std::boxed::Box<image::buffer_::ImageBuffer<image::color::Bgr<u8>, &[u8]>>` here
     |        the method is available for `std::sync::Arc<image::buffer_::ImageBuffer<image::color::Bgr<u8>, &[u8]>>` here
     |        the method is available for `std::rc::Rc<image::buffer_::ImageBuffer<image::color::Bgr<u8>, &[u8]>>` here
     |
     = help: items from traits can only be used if the trait is in scope
     = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
             `use image::buffer_::ConvertBuffer;`

error[E0599]: no method named `convert` found for struct `image::buffer_::ImageBuffer<image::color::Bgra<u8>, &[u8]>` in the current scope
    --> /home/sigkill/.cargo/registry/src/github.com-1ecc6299db9ec823/realsense-rust-0.3.0/src/base.rs:113:70
     |
113  |             Rs2Image::Bgra8(image) => DynamicImage::ImageBgra8(image.convert()),
     |                                                                      ^^^^^^^ method not found in `image::buffer_::ImageBuffer<image::color::Bgra<u8>, &[u8]>`
     | 
    ::: /home/sigkill/.cargo/registry/src/github.com-1ecc6299db9ec823/image-0.23.6/./src/buffer.rs:1059:8
     |
1059 |     fn convert(&self) -> T;
     |        -------
     |        |
     |        the method is available for `std::boxed::Box<image::buffer_::ImageBuffer<image::color::Bgra<u8>, &[u8]>>` here
     |        the method is available for `std::sync::Arc<image::buffer_::ImageBuffer<image::color::Bgra<u8>, &[u8]>>` here
     |        the method is available for `std::rc::Rc<image::buffer_::ImageBuffer<image::color::Bgra<u8>, &[u8]>>` here
     |
     = help: items from traits can only be used if the trait is in scope
     = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
             `use image::buffer_::ConvertBuffer;`

error[E0599]: no method named `convert` found for struct `image::buffer_::ImageBuffer<image::color::Rgb<u8>, &[u8]>` in the current scope
    --> /home/sigkill/.cargo/registry/src/github.com-1ecc6299db9ec823/realsense-rust-0.3.0/src/base.rs:114:68
     |
114  |             Rs2Image::Rgb8(image) => DynamicImage::ImageRgb8(image.convert()),
     |                                                                    ^^^^^^^ method not found in `image::buffer_::ImageBuffer<image::color::Rgb<u8>, &[u8]>`
     | 
    ::: /home/sigkill/.cargo/registry/src/github.com-1ecc6299db9ec823/image-0.23.6/./src/buffer.rs:1059:8
     |
1059 |     fn convert(&self) -> T;
     |        -------
     |        |
     |        the method is available for `std::boxed::Box<image::buffer_::ImageBuffer<image::color::Rgb<u8>, &[u8]>>` here
     |        the method is available for `std::sync::Arc<image::buffer_::ImageBuffer<image::color::Rgb<u8>, &[u8]>>` here
     |        the method is available for `std::rc::Rc<image::buffer_::ImageBuffer<image::color::Rgb<u8>, &[u8]>>` here
     |
     = help: items from traits can only be used if the trait is in scope
     = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
             `use image::buffer_::ConvertBuffer;`

error[E0599]: no method named `convert` found for struct `image::buffer_::ImageBuffer<image::color::Rgba<u8>, &[u8]>` in the current scope
    --> /home/sigkill/.cargo/registry/src/github.com-1ecc6299db9ec823/realsense-rust-0.3.0/src/base.rs:115:70
     |
115  |             Rs2Image::Rgba8(image) => DynamicImage::ImageRgba8(image.convert()),
     |                                                                      ^^^^^^^ method not found in `image::buffer_::ImageBuffer<image::color::Rgba<u8>, &[u8]>`
     | 
    ::: /home/sigkill/.cargo/registry/src/github.com-1ecc6299db9ec823/image-0.23.6/./src/buffer.rs:1059:8
     |
1059 |     fn convert(&self) -> T;
     |        -------
     |        |
     |        the method is available for `std::boxed::Box<image::buffer_::ImageBuffer<image::color::Rgba<u8>, &[u8]>>` here
     |        the method is available for `std::sync::Arc<image::buffer_::ImageBuffer<image::color::Rgba<u8>, &[u8]>>` here
     |        the method is available for `std::rc::Rc<image::buffer_::ImageBuffer<image::color::Rgba<u8>, &[u8]>>` here
     |
     = help: items from traits can only be used if the trait is in scope
     = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
             `use image::buffer_::ConvertBuffer;`

error[E0599]: no method named `convert` found for struct `image::buffer_::ImageBuffer<image::color::Luma<u16>, &[u16]>` in the current scope
    --> /home/sigkill/.cargo/registry/src/github.com-1ecc6299db9ec823/realsense-rust-0.3.0/src/base.rs:116:72
     |
116  |             Rs2Image::Luma16(image) => DynamicImage::ImageLuma16(image.convert()),
     |                                                                        ^^^^^^^ method not found in `image::buffer_::ImageBuffer<image::color::Luma<u16>, &[u16]>`
     | 
    ::: /home/sigkill/.cargo/registry/src/github.com-1ecc6299db9ec823/image-0.23.6/./src/buffer.rs:1059:8
     |
1059 |     fn convert(&self) -> T;
     |        -------
     |        |
     |        the method is available for `std::boxed::Box<image::buffer_::ImageBuffer<image::color::Luma<u16>, &[u16]>>` here
     |        the method is available for `std::sync::Arc<image::buffer_::ImageBuffer<image::color::Luma<u16>, &[u16]>>` here
     |        the method is available for `std::rc::Rc<image::buffer_::ImageBuffer<image::color::Luma<u16>, &[u16]>>` here
     |
     = help: items from traits can only be used if the trait is in scope
     = note: the following trait is implemented but not in scope; perhaps add a `use` for it:
             `use image::buffer_::ConvertBuffer;`

error: aborting due to 6 previous errors

Some errors have detailed explanations: E0432, E0599.
For more information about an error, try `rustc --explain E0432`.
error: could not compile `realsense-rust`.

To learn more, run the command again with --verbose.

```

**Comment by ed-aka-yaboi** [comment]
Here is the output of less /proc/cpuinfo for the first processor
```
processor       : 0
vendor_id       : GenuineIntel
cpu family      : 6
model           : 158
model name      : Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz
stepping        : 9
microcode       : 0xd6
cpu MHz         : 900.010
cache size      : 6144 KB
physical id     : 0
siblings        : 8
core id         : 0
cpu cores       : 4
apicid          : 0
initial apicid  : 0
fpu             : yes
fpu_exception   : yes
cpuid level     : 22
wp              : yes
flags           : fpu vme de pse tsc msr pae mce cx8 apic sep mtrr pge mca cmov 
pat pse36 clflush dts acpi mmx fxsr sse sse2 ss ht tm pbe syscall nx pdpe1gb rdt
scp lm constant_tsc art arch_perfmon pebs bts rep_good nopl xtopology nonstop_ts
c cpuid aperfmperf pni pclmulqdq dtes64 monitor ds_cpl vmx est tm2 ssse3 sdbg fm
a cx16 xtpr pdcm pcid sse4_1 sse4_2 x2apic movbe popcnt tsc_deadline_timer aes x
save avx f16c rdrand lahf_lm abm 3dnowprefetch cpuid_fault epb invpcid_single pt
i ssbd ibrs ibpb stibp tpr_shadow vnmi flexpriority ept vpid ept_ad fsgsbase tsc
_adjust bmi1 avx2 smep bmi2 erms invpcid mpx rdseed adx smap clflushopt intel_pt
 xsaveopt xsavec xgetbv1 xsaves dtherm ida arat pln pts hwp hwp_notify hwp_act_w
indow hwp_epp md_clear flush_l1d
bugs            : cpu_meltdown spectre_v1 spectre_v2 spec_store_bypass l1tf mds 
swapgs itlb_multihit srbds
bogomips        : 5599.85
clflush size    : 64
cache_alignment : 64
address sizes   : 39 bits physical, 48 bits virtual
power management:
```

@jerry73204 what other information do you need for the repro?

**Comment by jerry73204** [comment]
v0.3.0 has an error on using old path in image crate. It's now fixed in 0.3.2 and now the clippy could be happy now.

**Comment by ed-aka-yaboi** [comment]
can confirm it now builds and runs
